scripts/eurostat/life_expectancy_by_age_gender/generate_mcf.py

```diff
@@ -0,0 +1,102 @@
+import sys
+import pandas as pd
+import re
+
+def convert_range(s_input):
+    """convert range values from the format in StatVars to the format as 
+        QuantityRange in DataCommons"""
+    if 'OrMore' in s_input:
+        match = re.match(r'(\d+)OrMore([a-zA-Z]+)', s_input)
+        num = match.group(1)
+        unit = match.group(2)
+        return '[{} {} -]'.format(unit, num)
+    elif 'Upto' in s_input:
+        match = re.match(r'Upto(\d+)([a-zA-Z]+)', s_input)
+        num = match.group(1)
+        unit = match.group(2)
+        return '[{} - {}]'.format(unit, num)
+    elif 'To' in s_input:
+        match = re.match(r'(\d+)To(\d+)([a-zA-Z]+)', s_input)
+        num1 = match.group(1)
+        num2 = match.group(2)
+        unit = match.group(3)
+        return '[{} {} {}]'.format(unit, num1, num2)
+    else:
+        match = re.match(r'(\d+)([a-zA-Z]+)', s_input)
+        num = match.group(1)
+        unit = match.group(2)
+        return '[{} {}]'.format(unit, num)
+  
+def generate_statvar(statvars, path):
+    """generate the statvars from the list of StatVar names"""
+    by_age_template = ('Node: dcid:{stat_var}\n'
+                               'typeOf: dcs:StatisticalVariable\n'
+                               'populationType: schema:Person\n'
+                               'age: {age}\n'
+                               'measuredProperty: dcs:lifeExpectancy\n'
+                               'statType: dcs:measuredValue\n\n')
+    by_age_gender_template = by_age_template[:-1] + 'gender: schema:{gender}\n\n'
+
+    with open(path, 'w') as f_out:
+        for stat_var in statvars:
+            keys = stat_var.split('_')
+            age = convert_range(keys[2])
+            if len(keys) == 3:
```

Comment:
Add a comment for why 3? Or the expected structure for stat_var before split, and perhaps a test to make sure stat_var follows one of two expected formats.

Reply:
Added the comment and raise error if the stat_var is not in one of these two cases.